### ISSUE TITLE
Solution for the "inProcessTabChildGlobal leak".

### ISSUE THREAD

**Issue by f3Z0zRM** [MoonchildProductions/Basilisk#17]
My impression is that pull requests are the more regular way of doing this sort of thing however I have practically no experience with git and the tutorials for forking etc make it look unreasonably effort intensive for this given that this is a super trivial fix that just reverts an apparently unintentional deletion with a single line. Thank you for your understanding and if you tell me I must do the fork etc then I will, at some point.

To understand the issue at hand first look at https://github.com/MoonchildProductions/Basilisk/commit/f108a5ecca8b24c8cefb10d5f00670a36cc7ea16#diff-269db4162215802fabafaf8a361e5628 (AKA "Remove webextensions conditional code from Basilisk") and note that "RefreshBlocker.uninit();" is deleted along with the web extension cruft. Next look at the version of the file that changed there as of this writing: https://github.com/MoonchildProductions/Basilisk/blob/7ab9bf71dfa7fdfba2520d1538e6ac132ebcbf4f/basilisk/base/content/tab-content.js and note that RefreshBlocker is still there along with RefreshBlocker.init(). Next check out https://bugzilla.mozilla.org/show_bug.cgi?id=1247100 and the comment associated with RefreshBlocker and note that RefreshBlocker does not have an apparent connection to webextensions and probably still does something or other.

RefreshBlocker is an observer that holds references that keep the inProcessTabChildGlobal for the tab associated with it in memory(at least) and if it is not uninit'ed then RefreshBlocker and things it references are held in memory due to the "Services.prefs.addObserver(this.PREF, this, false);" call which causes a reference to RefreshBlocker to be kept by the pref service.

With the current state of affairs memory is leaked with every tab opened.

In conclusion: anybody in a position to do so, please put back the RefreshBlocker.uninit(); on unload. I have tried this personally for a few days now without problems and verified that this results in proper removal of inProcessTabChildGlobals from about:memory.

Thank you again for your understanding.

**Comment by wolfbeast** [comment]
If forking, patching and making a PR is too involved, at least provide a diff that could be applied. Just going by a description makes this bumped to a backburner level of attention, unfortunately.
